contracts/SwapRBTC.sol

```diff
@@ -128,6 +129,24 @@ contract SwapRBTC is Initializable, OwnableUpgradeable, ISwapRBTC, IERC777Recipi
     return amount;
   }
 
+  function swapRBTCtoSideTokenBtc(uint256 amount, address sideTokenBtcContract) external payable override returns (uint256) {
+    require(enumerableSideTokenBtc.contains(sideTokenBtcContract), "SwapRBTC: Side Token not found");
+    ISideToken sideToken = ISideToken(sideTokenBtcContract);
+    address sender = _msgSender();
+    
+    require(address(this).balance >= amount, "SwapRBTC: amount > balance");
+    require(sideToken.balanceOf(address(this)) >= amount, "SwapRBTC: not enough balance");
+    require(balance[msg.sender] >= amount, "SwapRBTC: sender not enough balance");
```

Comment:
please use the "sender" var instead, which points to _msgSender(). _msgSender() is a replacement for msg.sender. For regular transactions it returns msg.sender and for meta transactions it can be used to return the end user (rather than the relayer).